I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


File "game/base_of_operations.rpy", line 23: end of line expected.
    "[expedition_names[1]]": if False:
                             ^

Ren'Py Version: Ren'Py 8.4.1.25072401
Thu Sep 18 14:23:29 2025
